src/main/java/org/opentripplanner/graph_builder/module/ned/NEDDownloader.java

```diff
@@ -545,7 +545,7 @@ public String getPrefix(String namespace) {
         }
     }
 
-    public Iterator<?> getPrefixes(String namespace) {
+    public Iterator getPrefixes(String namespace) {
```

Comment:
Is this change actually needed to make things build/tests pass?

Reply:
@landonreed - JDK version related. 

This fails:
Project SDK: OpenJDK 11
Module (otp): Oracle JDK 1.8_191

This works:
Project SDK: Oracle JDK 1.8_191
Module (otp): Oracle JDK 1.8_191

I'm not sure why it is not using the module specific JDK. Anyway, assuming this project is still 1.8, I have removed this alteration.